pkg/operatorsource/purging.go

```diff
@@ -56,16 +55,6 @@ func (r *purgingReconciler) Reconcile(ctx context.Context, in *v1alpha1.Operator
 
 	r.datastore.RemoveOperatorSource(in.GetUID())
 
-	builder := &CatalogSourceConfigBuilder{}
-	csc := builder.WithTypeMeta().
-		WithNamespacedName(in.Namespace, in.Name).
-		CatalogSourceConfig()
-
-	if err = r.client.Delete(ctx, csc); err != nil && !k8s_errors.IsNotFound(err) {
-		nextPhase = phase.GetNextWithMessage(phase.OperatorSourcePurging, err.Error())
-		return
-	}
```

Comment:
Can you update the comments in this file to reflect that we are no longer actually "purging" the operator source object itself?

Reply:
This should also be reflected in log statement which says `Purging all resource(s)`

Reply:
Updated the log statement to clarify what has been purged.